### PR TITLE
Update PBS Pro version to 19.1.2

### PR DESCRIPTION
**Pull request by bremanandjk**
#### Describe Bug or Feature
Update PBS Pro version to 19.1.2

#### Describe Your Change

Changed below files.
1) configure.ac
2) INSTALL 
3) pbspro.spec

#### Link to Design Doc

NA

#### Attach Test and Valgrind Logs/Output

NA

### CODE REVIEW

File: pbspro.spec [hunk 1]
```diff
@@ -513,7 +513,6 @@ fi
 %exclude %{pbs_prefix}/lib*/python/pbs_bootcheck*
 %exclude %{pbs_prefix}/libexec/install_db
 %exclude %{pbs_prefix}/libexec/pbs_habitat
-%exclude %{pbs_prefix}/libexec/pbs_schema_upgrade
```
Comment (agrawalravi90) [inline pbspro.spec]: why did you remove this?

**Comment by bremanandjk** [comment]
> Minor comment. Also, could you please run qstat --version to show how the new version? that should be enough proof of testing. Thanks.

we can consider below "make dist" also as a proof that its using correct version to generate a *.tar.gz file.

```
[root@physics pbspro]# ls
aclocal.m4      buildutils     configure     doc      libtool  Makefile     PBS_License.txt       pbspro.spec     README.md  valgrind.supp
autogen.sh      config.log     configure.ac  docker   LICENSE  Makefile.am  **pbspro-19.1.2.tar.gz**  pbspro.spec.in  src        win_configure
autom4te.cache  config.status  COPYRIGHT     INSTALL  m4       Makefile.in  pbspro-rpmlintrc      README          test
```
And qstat --version o/p:
root@physics pbspro]# qstat --version
pbs_version = 19.1.2

File: pbspro.spec (hunk 1 above)
Comment (bremanandjk) [reply pbspro.spec]: This change was added only in pbspro.spec file as part of 1f8828f022bfd7e08cf3a53c55902c455ad8d74d, I have just synced with the latest changes by running autogen.sh. 

The right fix would be adding the change in the proper file "pbspro.spec.in". I have added it now.

Thanks for your comment, I have pushed it, plz review.

File: pbspro.spec.in [hunk 2]
```diff
@@ -513,6 +513,7 @@ fi
 %exclude %{pbs_prefix}/lib*/python/pbs_bootcheck*
 %exclude %{pbs_prefix}/libexec/install_db
 %exclude %{pbs_prefix}/libexec/pbs_habitat
+%exclude %{pbs_prefix}/libexec/pbs_schema_upgrade
```
Comment (agrawalravi90) [inline pbspro.spec.in]: So, in master branch, there's a %exclude for pbs_schema_upgrade under pbs_execution as well. Do you think we should also add it there?

TBH, since this is kind of out of scope for your PR anyways, I would just leave these files as it is.

Comment (bremanandjk) [reply pbspro.spec.in]: ok, I agree
I have removed out of scope file changes,

**Comment by bremanandjk** [comment]
Thanks for your approvals, @agrawalravi90 
I will squash the commits and prepare to merge.

**Comment by agrawalravi90** [comment]
you need one more sign-off
